sdk-setup/src/mb2

```diff
@@ -1027,14 +1027,14 @@ get_device() {
 rsync_as() {
     local user="$1";shift
     local key=$(get_shared_dir)/${device_sshkeypath}/$user
-    [[ -f "$key" ]] || fatal "No key for user '$user' on '$device_name' given in devices.xml (rsync '$key')"
+    [[ -f "$key" ]] || fatal "SSH key not configured for user '$user' on '$device_name'"
```

Comment:
Shouldn't we give the user some hint about how to configure the key?

Reply:
I though about that too, but didn't like it. Not only that specifically in case of mb2 there are three frontedns (including mb2 itself), each deserving different handling. In general hints hardcoded in code are not ideal. It is better to issue a brief but distinguishable error message and have an answer elsewhere easy to look up based on the error message (docs, TJC, ...) - such an information is much easier to expand, improve, correct, ..., without all the integration hassle.